numpyro/distributions/continuous.py

```diff
@@ -490,6 +490,51 @@ def icdf(self, q):
         return self.loc - self.scale * jnp.log(-jnp.log(q))
 
 
+class Gompertz(Distribution):
+    arg_constraints = {
+        "loc": constraints.positive,
+        "shape": constraints.positive,
+    }
+    support = constraints.positive
+    reparametrized_params = ["loc", "shape"]
+
+    def __init__(self, loc=1.0, shape=1.0, validate_args=None):
```

Comment:
In NumPyro, it is more common to use names `mean` and `concentration` (loc is typically used for loc-scale families) -see e.g. [BetaProportion](https://github.com/pyro-ppl/numpyro/blob/master/numpyro/distributions/continuous.py#L1556). Is the default value 1., 1. commonly used (I guess we don't need to specify default values for those parameters.)? You can use `concentration` and `rate` parameterizations to match Gamma distribution (which is related to this distribution) API. Then you can set the default value `rate=1.`.

Reply:
I went with `rate` and `concentration`.

Reply:
I like those parameters. Could you swap the order and set `rate=1.` by default? (this is to match Gamma distribution)

Reply:
taken care of